go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/mattermost/bifrost
 
-go 1.15
+go 1.16
```

Comment:
why not 1.17? Are there re any clients that consume this that they are using go 1.16?

Reply:
It returned error that is not supported. I am not sure if I did something wrong though.